dialect_redshift.go

```diff
@@ -0,0 +1,19 @@
+package gorp
+
+type RedshiftDialect struct {
+	PostgresDialect
+}
+
+func (d RedshiftDialect) InsertAutoIncrToTarget(exec SqlExecutor, insertSql string, target interface{}, params ...interface{}) error {
+	rows, err := exec.Query(insertSql, params...)
+	defer rows.Close()
+	return err
+}
```

Comment:
For anyone else curious, this is the postgres dialect method that this is based on:
https://github.com/clyphub/gillnet/blob/5b51c50532ebf425d76a12977b9dc671927d8039/go/src/clypd/vendor/github.com/clyphub/gorp/dialect_postgres.go#L106-L123